### PR TITLE
Add gbage

### LINKED ISSUE #43: Printing branches and their age.
Found this command on the stackoverlflow : 
`git for-each-ref --sort=committerdate refs/heads/ --format='%(HEAD) %(color:yellow)%(refname:short)%(color:reset) - %(color:red)%(objectname:short)%(color:reset) - %(contents:subject) - %(authorname) (%(color:green)%(committerdate:relative)%(color:reset))'
`

Outpu example:

which shows all branches and their age. I find this very useful. Can we have this in this tool as well? 🙏🏻 

Thanks.

### PR DESCRIPTION
**Pull request by MartinDelille**
@axell Here is a working implementation to fix #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | gban         | `git branch -a -v --no-merged`                       |
 | gbda         | delete all branches merged in current HEAD           |
 | ggsup        | git set upstream to origin/_current-branch_          |
+| gbage        | `git for-each-ref --sort=committerdate refs/heads/ --format='%(HEAD) %(color:yellow)%(refname:short)%(color:reset) - %(color:red)%(objectname:short)%(color:reset) - %(contents:subject) - %(authorname) (%(color:green)%(committerdate:relative)%(color:reset))'` |
```
Comment (jhillyerd) [inline README.md:41]: I don't think this description is easily readable, let's just summarize what it does.

File: init.fish [hunk 2]
```diff
@@ -10,9 +10,10 @@ abbr -a g          git
 abbr -a ga         git add
 abbr -a gaa        git add --all
 abbr -a gapa       git add --patch
+abbr -a gb         git branch -vv
 abbr -a gba        git branch -a -v
 abbr -a gban       git branch -a -v --no-merged
-abbr -a gb         git branch -vv
+abbr -a gbage      "git for-each-ref --sort=committerdate refs/heads/ --format=\"%(HEAD) %(color:yellow)%(refname:short)%(color:reset) - %(color:red)%(objectname:short)%(color:reset) - %(contents:subject) - %(authorname) (%(color:green)%(committerdate:relative)%(color:reset))\""
```
Comment (jhillyerd) [inline init.fish:16]: Abbreviations are best for shorter commands that one might edit or append to after typing.  This looks like it belongs in a function to me.

**Review by jhillyerd — CHANGES_REQUESTED** [review]
Note that GitHub is warning of a merge conflict, please rebase or use a clean branch.

File: README.md (hunk 1 above)
Comment (MartinDelille) [reply README.md]: My local repo was indeed old and I didn't noticed I could simply put a description there => Done!

File: init.fish (hunk 2 above)
Comment (MartinDelille) [reply init.fish]: Of course. Fixed!

**Comment by jhillyerd** [comment]
Looks good, thank you!

**Pull request merged**